R/creatinine_scatter_charts.R

```diff
@@ -96,9 +108,13 @@ draw_creatinine_scatter <- function(df, settings) {
   ggply$x$data[[6]]$hoverinfo <- "none"
   
   ggply$x$data[[7]]$hoverinfo <- "none"
- 
-  ggply
- 
+  
+  if (animate == "on") {
+    ggply %>%  animation_opts(frame = 1000, transition = 500, redraw = FALSE)
```

Comment:
Could we have this be a setting for a user? When I was playing around with it I thought the animation was slow. I know it might clutter up the design a bit as well so I'm open to other ideas.

Reply:
could you tell me more about this - want to make sure I'm following

Reply:
I was thinking of an argument like animation_transition_time then:
```
    ggply %>%  animation_opts(frame = animation_transition_time + 500, transition = animation_transition_time , redraw = FALSE)
```
And add an input for the user to select animation time to control speed

Reply:
ahhhhhh i see thats a really cool idea, maybe I'll throw that in there real quick